### PR TITLE
feat(schemas): Moving schemas to a new repo

### PR DESCRIPTION
**Pull request by kpears201**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@babel/preset-env": "^7.15.0",
+    "@firebolt-js/schemas": "file:../firebolt-schemas",
```
Comment (kpears201) [inline package.json:42]: Will change once new package is released to npm

File: util/declarations/index.mjs [hunk 2]
```diff
@@ -44,7 +44,7 @@ const run = ({
   // Important file/directory locations
   const declarationsFile = path.join(outputFile)
   const schemasFolder = path.join(srcFolderArg, 'schemas')
-  const sharedSchemasFolder = path.join(__dirname, '..', '..', 'src', 'schemas')
+  const sharedSchemasFolder = path.join(__dirname, '..', '..', 'node_modules', '@firebolt-js', 'schemas', 'src', 'schemas')
```
Comment (jlacivita) [inline util/declarations/index.mjs:47]: We should make a todo to parameterize this, so the sharedSchemasFolder is passed in. We shouldn't assume everyone using the tool has this dependency.

**Review by jlacivita — DISMISSED** [review]
LGTM

File: util/declarations/index.mjs [hunk 3]
```diff
@@ -39,12 +39,13 @@ const __dirname = url.fileURLToPath(new URL('.', import.meta.url))
 // destructure well-known cli args and alias to variables expected by script
 const run = ({
   source: srcFolderArg,
+  'shared-schemas': sharedSchemasFolderArg,
   output: outputFile
 }) => {
   // Important file/directory locations
   const declarationsFile = path.join(outputFile)
   const schemasFolder = path.join(srcFolderArg, 'schemas')
-  const sharedSchemasFolder = path.join(__dirname, '..', '..', 'src', 'schemas')
```
Comment (jlacivita) [inline util/declarations/index.mjs:47]: love getting rid of '../..' from code!

**Review by jlacivita — DISMISSED** [review]
LGTM

**Review by cletustboone — COMMENTED** [review]
This makes more sense after yesterday's discussion.

**Pull request merged**